packages/fluentui/accessibility/src/behaviors/Table/tableBehavior.ts

```diff
@@ -19,4 +19,8 @@ const tableBehavior: Accessibility = props => ({
   },
 });
 
+export type TableBehaviorProps = {
```

Comment:
Please why do we need the prop for tableRow in the Table behavior?

Reply:
It's being used on the `childBehaviors`

Reply:
Solved